index.bs

```diff
@@ -5654,6 +5665,9 @@ To <dfn>retrieve a value from an object store</dfn> with
 To <dfn>retrieve multiple values from an object
 store</dfn> with |targetRealm|, |store|, |range| and optional |count|, run these steps:
 
+NOTE:
```

Comment:
Same as above.

Reply:
Updated. I've specified the return type as `sequence<any>` here since that is what we return from the last step of this algorithm. Perhaps
> ... a sequence of ECMAScript values or ...

would be better (and aligned with the return type of the single value retrieval above), but I'm not aware of how implicit the equivalence between `any` and `ECMAScript value` is. Do LMK if there is a common term we can use for both these algorithms.